softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/factory/AbstractFactory.java

```diff
@@ -0,0 +1,12 @@
+package com.softgroup.common.router.api.factory;
+
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.router.api.Handler;
+
+/**
+ * Created by polev on 07.04.2017.
+ */
+public interface AbstractFactory<T extends Handler> {
```

Comment:
@MariiaPolova interface can not be abstract. And what is the difference between this interface and Factory interface ?